diff --git a/node_modules/react-login-github/src/LoginGithub.js b/node_modules/react-login-github/src/LoginGithub.js
index b301019..d98c1cc 100644
--- a/node_modules/react-login-github/src/LoginGithub.js
+++ b/node_modules/react-login-github/src/LoginGithub.js
@@ -21,7 +21,7 @@ class LoginGithub extends Component {
   }
 
   static defaultProps = {
-    buttonText: 'Sign in with GitHub',
+    buttonText: 'HELLO',
     redirectUri: '',
     scope: 'user:email',
     popupHeight: 650,
